### ISSUE TITLE
convLSTM

### ISSUE THREAD

**Issue by 18031348734** [RobinLuoNanjing/air_pollutants_prediction_lstm#2]
您好，我是一名研二学生，想通过您的代码学习convLSTM，但是在进行训练的时候，运行“train_model('nox',model_nox)”总是出现如下错误："Can't call numpy() on Tensor that requires grad. Use tensor.detach().numpy() instead."可以向您请教一下是什么原因吗，谢谢您啦

**Comment by SiYXiN** [comment]
> 您好，我是一名研二学生，想通过您的代码学习convLSTM，但是在进行训练的时候，运行“train_model('nox',model_nox)”总是出现如下错误："Can't call numpy() on Tensor that requires grad. Use tensor.detach().numpy() instead."可以向您请教一下是什么原因吗，谢谢您啦

你好，我想问一下你前面的问题解决了么

**Comment by keyboardman42** [comment]
这个错误是由于你试图将一个带有梯度信息的Tensor类型变量转换为numpy数组导致的
如果你有一个带梯度信息的Tensor类型变量t，你希望将其转换为ndarray，可以通过以下代码实现
array = t.detach().numpy()
